### PR TITLE
Interceptor

### PR DESCRIPTION
**Pull request by claireriley**
Interceptor redraft—let me know what you think, re-styled and reworded a bit. Had trouble using active voice for some of the more prescriptive sections.

@abhijitkane could you particularly check the section "Troubleshooting tips"? That was one that I had to make a few assumptions on as it was a bit unclear. I did my best to maintain the original meaning of all sentences/sections here, just moved some words around.

### CODE REVIEW

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+You can retrieve cookies from any domain and use them in your Postman requests by enabling **Capture cookies**. You can then add your domain, or multiple, with **Add Domain**.
+
+[![interceptor on](https://assets.postman.com/postman-docs/Interceptor-ex1.gif)](https://assets.postman.com/postman-docs/Interceptor-ex1.gif)
+
+Interceptor will sync all cookies for that domain from the browser. It will also sync cookies for all subdomains.
+
+> For example, adding the domain `facebook.com` will also sync cookies from `m.facebook.com`.
+
+Interceptor keeps cookies for a fixed set of domains in sync, updating Postman from the browser when cookies update, including deletions. Due to this, you can use authentication sessions in your browser to make API calls in Postman. You will not be able to save these to Postman’s history.
```
Comment (abhijitkane) [inline src/pages/docs/postman/sending-api-requests/interceptor.md]: @claireriley I wouldn't want to restrict this to authentication sessions - just "you can use browser sessions to make API calls in Postman" works.

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+Once the **Interceptor connected** status is green, you can add URL or HTTP method filters by clicking **Show additional filters**.
+
+[![capturing requests](https://assets.postman.com/postman-docs/interceptor-ex2.jpg)](https://assets.postman.com/postman-docs/interceptor-ex2.jpg)
+
+## How it works
+
+Interceptor works by leveraging a Google Chrome feature and the installation process differs according to your operating system.
+
+Postman's native app requires Chrome's [Native Messaging](https://developer.chrome.com/extensions/nativeMessaging) feature. This is an independent executable and will be installed on your machine.
```
Comment (abhijitkane) [inline src/pages/docs/postman/sending-api-requests/interceptor.md]: This requires an independent executable

File: src/pages/docs/postman/sending-api-requests/capturing-http-requests.md [hunk 3]
```diff
@@ -31,16 +31,16 @@ warning: false
...
+If you are using APIs to build client-side applications - mobile apps, websites or desktop applications - you might want to see the actual HTTP request traffic that is being sent and received in the application. In some cases, you might discover APIs that are not even documented. Postman gives you tools to see and capture this network traffic easily. You can use the built-in proxy in the Postman native apps or use the [Interceptor extension](/docs/postman/sending-api-requests/interceptor/) for the Postman app.
 
 **Note:** For the Postman native apps, request captures over HTTPS will not work if the website has HSTS enabled. Most websites have this check in place.
 
 You can capture the HTTP requests using the following two ways:
 
-* [Using the Postman built-in proxy](#using-the-postman-built-in-proxy)
-* [Using the Interceptor](#using-the-interceptor)
+* [Postman's built-in proxy](#using-the-postman-built-in-proxy)
```
Comment (SueSmith) [inline src/pages/docs/postman/sending-api-requests/capturing-http-requests.md]: Am always a bit wary of "Postman's" but not a huge deal..

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+  4. **Add an executable**: This is a new process started by Chrome when required by the Interceptor extension. For Windows and Linux, this is a self-contained binary that is approximately 40MB in size. For MacOS, this is a 33KB JavaScript file. This file must be executable. This executable will be installed in a location dependent on your operating system:
+    * **MacOS**: `$HOME/.postman/InterceptorBridge`
+    * **Windows**: `%USERPROFILE%/.postman/InterceptorBridge`
+    * **Linux**: `$HOME/.postman/InterceptorBridge`
+
+### Troubleshooting tips
+
+You can find the current status of your Interceptor integration by selecting the Interceptor satellite icon in the upper-right corner of the app, then navigating to **Cookies**:
+
+* **Connected**: You're good to move forward with using Interceptor.
```
Comment (SueSmith) [inline src/pages/docs/postman/sending-api-requests/interceptor.md]: I'd maybe go for something like "You can proceed..."

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+* **Not connected**: You'll want to make sure Interceptor is installed correctly—refer to [Installing interceptor](#installing-interceptor).
```
Comment (SueSmith) [inline src/pages/docs/postman/sending-api-requests/interceptor.md]: Maybe just "Ensure Interceptor is installed..."

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+* **Dependencies not installed**: You will be guided by the Postman UI to install the required dependencies.
```
Comment (SueSmith) [inline src/pages/docs/postman/sending-api-requests/interceptor.md]: "Postman will indicate..." maybe?

**Comment by claireriley** [comment]
Add https://postmanlabs.atlassian.net/wiki/spaces/PRODOPS/pages/1722320000/Reset+API+for+Native+App+Interceptor+Integration to troubleshooting

**Pull request merged**